after/syntax/javascript.vim

```diff
@@ -117,12 +117,12 @@ syn match styledPrefix "\.\<extend\>"
 " annotations (delimited by brackets (e.g. "<>")) between `styled` and
 " the function call parenthesis
 syn match styledTypescriptPrefix
-      \ "\<styled\><\%(\k\|'\|\"\|`\|,\|\s\)\+>(\%('\k\+'\|\"\k\+\"\|`\k\+`\))"
+      \ "\<styled\><.\+>(.\+)"
```

Comment:
Is there a reason, why you remove the specificy of this match? Matching by `.` can be really expensive.

Reply:
I had trouble getting it to work, but now I see that only the inline-interface is a problem. I'll need to look into these magic regexes a little more to see if I can solve it.

Reply:
I never directly answered, sorry.
Yeah, sure. If you need help or anything, just lemme know. Maybe if you know, what you want to regex-ify, just post it here and I can try to help regex-izise it. :smile: